include/deal.II/grid/tria_accessor.h

```diff
@@ -1627,7 +1626,13 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
    * triangulation.
    */
   void
-  set(const dealii::internal::TriangulationImplementation::TriaObject &o) const;
+  set(const std::initializer_list<int> &o) const;
+
+  /**
+   * The same as above but for `unsigned int`.
+   */
+  void
+  set(const std::initializer_list<unsigned int> &o) const;
```

Comment:
Ah, nice trick. I really struggled with all of the narrowing warnings and what to do about it :-)

Reply:
I really hate this (and the previous) hack. But, this was the approach to get as few narrowing warnings as possible.